pkg/kraken/rest.go

```diff
@@ -23,17 +24,21 @@ func NewRequest() *Request {
 	return &Request{
 		Request: &http.Request{
 			Method:     "GET",
-			Proto:      "HTTP/1.1",
-			ProtoMajor: 1,
-			ProtoMinor: 1,
+			Proto:      "HTTP/2",
+			ProtoMajor: 2,
+			ProtoMinor: 0,
```

Comment:
are these lines even needed? surely the client chooses HTTP/2 correctly when connecting via HTTPS?